src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/service/VisitService.kt

```diff
@@ -225,8 +227,8 @@ class VisitService(
     return Visit(
       offenderBooking = offenderBooking,
       visitDate = LocalDate.from(visitDto.startDateTime),
-      startTime = visitDto.startDateTime,
-      endTime = LocalDateTime.of(LocalDate.from(visitDto.startDateTime), visitDto.endTime),
+      startDateTime = visitDto.startDateTime,
+      endDateTime = LocalDateTime.of(LocalDate.from(visitDto.startDateTime), visitDto.endTime),
       visitType = visitType,
       visitStatus = visitStatusRepository.findById(VisitStatus.pk("SCH")).orElseThrow(),
```

Comment:
curious, not sure I understand this visitStatus, isn't it always returning scheduled?

Reply:
yeah - doesn't seem right.  I would have thought using the value passed by the sync service (albeit always "SCH" would be correct